src/components/view/sourcesView.ts

```diff
@@ -0,0 +1,74 @@
+import App from "../app/App";
+import Option from "../Dto/Option";
+import RequestParameters from "../Dto/RequestParameters";
+import SourceResult from "../Dto/SourceResult";
+import NewsResult from "../Dto/NewsResult";

```

Comment:
Use alias import for such files. It'll be an easy-to-read code
![image](https://user-images.githubusercontent.com/76782175/179418592-be0f14c2-8dda-4f0b-a74f-14c872e9e65b.png)